### DISCUSSION TITLE
Are all IT8951 Boards supported?

### DISCUSSION THREAD

**Discussion by DerPicknicker** [txoof/PaperPi#157] [Q&A]
Hello guys, 

Cool project - I wanted to know if all boards are supported. I think there are some other great boards available besides the waveshare ones. 

For example here: https://x.alibaba.com/Aw8htQ?ck=pdp

13,3 inch incl. Controller board for 230$ seems like an fair price. 

Open for your opinions or suggestions for sizes around 10-13"

**Comment by txoof** [comment]
That's a good question. So far, no one has tested on anything other than the waveshare kits. 

The IT8951 support comes via the fantastic work of [GregDMeyer](https://github.com/GregDMeyer) and the [IT8951 Module](https://github.com/GregDMeyer/IT8951).You'd need to check in with the folks over on that project to check if the IT8951 compatibility is universal, or limited just to the WaveShare boards. 

If you learn more, let me know and I'll update the documentation to provide that info too.

**Comment by DerPicknicker** [comment]
Hi,

What should we do now...? Should I create discussion in the other repo as well? 

This type would be an ideal panel for your project: https://www.panelook.com/ES133TT3_E%20Ink_13.3_EPD_overview_31869.html

High resolution - big size...

**Comment by txoof** [comment]
@DerPicknicker Yeah, It would be best to ask over at the GregDMeyer repo and then reference this discussion. I don't know much about the internals of that project. I follow discussions there and I'm happy to throw in with what I know. 

I have a wave share 9.7" that i really love; a 13.3" screen would be amazing, but I don't have EUR300 to drop on one right now.

**Comment by DerPicknicker** [comment]
Yeah the thing is why do the eink guys make it so complicated to find a panel + controller. I really can't understand
that. I found a shop which sells the 31" colored version for around 300€ which is really cheap. But without the matching controller it's totally useless. 


For example the ed113tc1 costs only 35€ but in the whole internet you cannot find a working controller due a wrong connector or missing configuration. Because each panel needs his own waveform. 

Sorry for the rant but it's really hard to love this technology...

**Reply by txoof** [reply]
A lot of what you are finding is intended for the professional market. The  boards that wave share et al provides are really just demo hardware that's provided to show the capability and functionality of the panels.

The cheap panels you see are meant to go in mass produced devices like the shelf tags, or e-readers. The manufacturer makes custom driver boards and software to run these to their own spec, they don't use the demo boards.

If you want to have a hobby play with these, you'll have to either use the waveshare stuff, or get to hacking and build your own hardware/firmware.
